scipy/stats/_morestats.py

```diff
@@ -2645,6 +2646,8 @@ def levene(*samples, center='median', proportiontocut=0.05):
     return LeveneResult(W, pval)
 
 
+@_deprecated("'binom_test' is deprecated in favour of"
+             "'binomtest' and will be removed in Scipy 1.11.0.")
```

Comment:
Missing a space here 
```suggestion
             " 'binomtest' and will be removed in Scipy 1.11.0.")
```